### ISSUE TITLE
[Question] Markdown: point a file with absolute path

### ISSUE THREAD

**Issue by rjshrjndrn** [vimwiki/vimwiki#746]
Hi,
Thank you for this awesome plugin.
I'm trying to figure out one thing but in vain.
`  * [Ansible refactor](/home/rajeshr/Documents/projects/public-devops/README.md)`
How to open the file, in markdown mode ?

**Comment by ranebrown** [comment]
Seems to be a bug, I'll try and look into it but I'm a bit short on time right now.

**Comment by ranebrown** [comment]
As a workaround you could put the below function in your `vimrc` and then open the absolute links with `[link](vfile:/absolute/path/here)`

```vim
function! VimwikiLinkHandler(link)
    let link = a:link
    if link =~# '^vfile:'
        let link = link[1:]
    else
        return 0
    endif
    let link_infos = vimwiki#base#resolve_link(link)
    if link_infos.filename ==? ''
        echomsg 'Vimwiki Error: Unable to resolve link!'
        return 0
    else
        exe 'edit' . fnameescape(link_infos.filename)
        return 1
    endif
endfunction
```

**Comment by rjshrjndrn** [comment]
It's working. Thank you.

**Comment by tinmarino** [comment]
In `function! vimwiki#base#resolve_link(link_text, ...)` to many changes are made to the `link_text` even if this one is absolute (in which case it should not change).
I tried as clean as I could for easy integration : PR #755

**Comment by justrajdeep** [comment]
I attempted to make `[//absolute_path]` modification to the plugin and created a PR#811.

**Comment by tinmarino** [comment]
Fixed: #811 with 
```
* [Ansible refactor](///home/rajeshr/Documents/projects/public-devops/README.md)
```
